core/tests/Test.Microsoft.Identity.LabInfrastructure/LabServiceApi.cs

```diff
@@ -82,7 +82,10 @@ private IUser GetUserFromLab(UserQueryParameters query)
                 throw new LabUserNotFoundException(query, "No lab user with specified parameters exists");
             }
 
-            LabUser user = JsonConvert.DeserializeObject<LabResponse>(result).Users;
+            LabResponse response = JsonConvert.DeserializeObject<LabResponse>(result);
+
+            LabUser user =  response.Users;
+            user.AppId = response.AppId;
```

Comment:
>user.AppId = response.AppId; [](start = 12, length = 28)

it's a bit odd that the appId is tied to the user. Having a parent object which holds a user and the app id feels more natural to me.